models/br_ibge_censo_2022/br_ibge_censo_2022__indigenas_populacao_residente_municipio.sql

```diff
@@ -0,0 +1,14 @@
+{{ config(alias='indigenas_populacao_residente_municipio',schema='br_ibge_censo_2022') }}
+WITH ibge as (
```

Comment:
Tem alguns id_municipios nulos. Além disso: 
- A variável `populacao_residente`  é igual a `pessoas_indigenas`? Vi que quando populacao_residente não é nula é igual a `pessoas_indigenas`. 
- Se for, acho que vale colocar na descrição da tabela

Reply:
é confuso, mas é assim mesmo. essa tabela no formado long ficaria mais "legível", mas a gnt deixou tudo em wide por conta do padrão da bd. A população residente sempre vai ser igual a pessoas indigenas nesses casos por conta da coluna `quesito_declaracao_indigena`, sempre que o individuo for indigena (i.e `quesito_declaracao_indigena == "
Cor ou raça indígena" ou "Se considera indígena"`) a populacao_residente vai ser igual a pessoas_indigenas, por conta dessa quebra. A informação de população residente ainda é importante para calcular o total de residentes no município. Mas vou add esse comentário na descrição da tabela. <img width="839" alt="Captura de Tela 2023-12-20 às 15 07 25" src="https://github.com/basedosdados/queries-basedosdados/assets/62671380/84c3a264-70aa-48e5-bdb9-0281bd0012a6"> @folhesgabriel